__tests__/cli.test.ts

```diff
@@ -238,6 +249,34 @@ describe('CLI', () => {
     expect(await cli.exitCode).toBe(0);
     process.env['TS_NODE_TYPE_CHECK'] = 'false';
   });
+
+  describe('testing a TLS site with self-signed cert', () => {
```

Comment:
nit: Would keep it smaller
```suggestion
  describe('TLS site with self-signed cert', () => {
```